server/src/config/dev.js

```diff
@@ -0,0 +1,6 @@
+export const config = {
+    secrets: {
+      jwt: 'mentor and stuff'

```

Comment:
I assume these 'secrets' are just development placeholders and are meant to be pushed up?

Reply:
yeah. These are local secrets.